src/traits.rs

```diff
@@ -136,6 +136,23 @@ where
 ///# Ok(())
 ///# }
 ///```
+///
+/// # Heap allocation
+///
+/// For systems where stack space is limited but heap space is available
+/// (i.e. ESP32 platforms with PSRAM) it's possible to allocate the display buffer
+/// in heap; the Displayxxxx:default() is implemented as always inline, so you can
+/// use Box::new to request heap space for the display buffer.
+///
+///```rust, no_run
```

Comment:
```suggestion
///```rust, no_run
///# use epd_waveshare::epd4in2::Display4in2;
///# use embedded_graphics_core::prelude::*;
```

Reply:
We need to add the necessary includes